src/com/strings/testStringsArray.java

```diff
@@ -0,0 +1,36 @@
+package com.strings;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class testStringsArray {
+
+
+        public static void main(String[] args) throws IOException {
```

Comment:
If the main method throws an exception, then the base class will handle it by issuing a stack trace to the console and ending the program, meaning your program would not have any reaction when input data is incorrect. Try to handle this exception in terms of your main method. Because IOException is checked and must be handled

Reply:
Added try and catch for invalid number format.